rust/src/measures/ffi.rs

```diff
@@ -340,3 +341,30 @@ impl<Q> Default for TypedMeasure<Q> {
 impl<Q> Measure for TypedMeasure<Q> {
     type Distance = Q;
 }
+
+#[bootstrap(
+    name = "new_privacy_profile",
+    features("contrib"),
```

Comment:
Add `honest-but-curious`? I feel like that's been standard for user plugins.

Reply:
good catch